app/model/Model.scala

```diff
@@ -30,6 +30,10 @@ import uk.gov.hmrc.auth.core.ConfidenceLevel
 import uk.gov.hmrc.http.SessionKeys
 import uk.gov.hmrc.play.json.Union
 
+// TODO: Is the default format for Joda correct for Gatekeeper?
+import play.api.libs.json.JodaReads._
+import play.api.libs.json.JodaWrites._
+
```

Comment:
Anybody got views on this?
Not sure just using Joda implicits will be correct in this context.